index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8"/>
+</head>
+<body>
+<video controls muted></video>
+<script>
+    const video = document.querySelector('video');
+    const assetURL = 'ws://localhost:9999/ws';
+    const mimeCodec = 'video/mp4; codecs="avc1.64001e, mp4a.40.2"';
```

Comment:
I'd double check the video codec to make sure it's not avc1.42C01F. That's what I got when I used the new FFMPEG settings.

Reply:
I will double check, but this is what mp4info returned for me. Maybe this differs per ffmpeg installation version/build flags etc? Homebrew default for me.